unlock-js/src/accounts.js

```diff
@@ -1,10 +1,12 @@
+import { walletEncryptionOptions } from './constants'
+
 const Wallet = require('ethers').Wallet
 
 export async function createAccountAndPasswordEncryptKey(password) {
```

Comment:
why don't we make this a parameter, and use the constants value as a default?

Reply:
I think that by having it only as a constant, we enforce some level of consistency so that there is only one place to set our standard account security. With an optional param I'd be concerned that sometimes we'd create accounts with the default level and other times with a custom one, even though all accounts should probably be the same.

Reply:
yes. The same thing will happen if we choose to change this later. If we were to change it, would that break decryption, or is this somehow encoded in the encryption metadata?

Reply:
I just added a test that illustrates that, but here's a more explicit example:
```javascript
let data = {
    id: "fb1280c0-d646-4e40-9550-7026b1be504a",
    address: "88a5c2d9919e46f883eb62f7b8dd9d0cc45bc290",
    Crypto: {
        kdfparams: {
            dklen: 32,
            p: 1,
            salt: "bbfa53547e3e3bfcc9786a2cbef8504a5031d82734ecef02153e29daeed658fd",
            r: 8,
            n: 262144 // <= This guy right here
        },
        kdf: "scrypt",
        ciphertext: "10adcc8bcaf49474c6710460e0dc974331f71ee4c7baa7314b4a23d25fd6c406",
        mac: "1cf53b5ae8d75f8c037b453e7c3c61b010225d916768a6b145adf5cf9cb3a703",
        cipher: "aes-128-ctr",
        cipherparams: {
            iv: "1dcdf13e49cea706994ed38804f6d171"
         }
    },
    "version" : 3
};
```

The information is included in the encrypted payload, so the decrypt method always has what it needs. This will allow us to boost this number for new accounts in the future (and re-encrypt existing accounts on first login after a change, if we want)

Reply:
appreciate the added detail, thank you